test/test_rpc.py

```diff
@@ -223,6 +230,41 @@ def test_py_function_exception(self):
             expected = "run_python_udf_internal caught exception: " + str(e)
         self.assertEqual(ret, expected)
 
+    @_wrap_with_rpc
+    def test_autograd_send_function(self):
+        dst_rank = (self.rank + 1) % self.world_size
+        with dist_autograd.context() as context_id:
+            t1 = torch.ones(3, 3, requires_grad=True)
+            t2 = torch.zeros(3, 3, requires_grad=True)
+            ret = dist.rpc('worker{}'.format(dst_rank), torch.add,
+                           args=(t1, t2))
+
+            # Get send function.
+            ctx = dist_autograd._current_context()
+            self.assertEqual(context_id, ctx._context_id())
+            send_functions = ctx._send_functions()
```

Comment:
Is this just for temporary test purpose, or do we plan to keep this API in the final version?